feat: Keyboard shortcut to delete current image and clear scene

lchauvin opened the pull request:
Add a keyboard shortcut to delete the current image (ctrl+d), and clear the whole scene (ctrl+w)

PaulHax commented on src/config.ts:
```diff
@@ -287,6 +287,9 @@ export const ACTION_TO_KEY = {
   decrementLabel: 'q',
   incrementLabel: 'w',
 
+  deleteCurrentImage: 'ctrl+d',
+  clearScene: 'ctrl+w',
```
These combos are used by Chrome too.  Any other ideas for defaults here?

https://support.google.com/chrome/answer/157179?hl=en&co=GENIE.Platform%3DDesktop#zippy=%2Ctab-window-shortcuts

lchauvin replied:
I was thinking that without a modifier key maybe people could press on it by accident (to write something) and close the scene or delete an image by accident

PaulHax replied:
hmm, your right, accidential would be sad.

lchauvin replied:
I put 'd' and 'k' for now. Do you want me to change for  'ctrl+/' and 'ctrl+.' ?

PaulHax replied:
No no, I thrashed around enought.  Lets see what @floryst thinks.

floryst replied:
I would rather have deletion shortcuts behind a modifier. This can either be ctrl or even just shift (e.g. `SHIFT-3`).

lchauvin replied:
I think I would go for ctrl instead. I'm thinking that Shift-1, etc... could be use to setup some presets later on (like Axial primary, Coronal primary, etc...)

PaulHax reviewed (approved):
Looks good!  Gracias.

floryst reviewed (changes requested):
Whatever your final keybinds are, please add them to https://github.com/Kitware/VolView/blob/main/docs/mouse_controls.md.

lchauvin commented:
> Whatever your final keybinds are, please add them to https://github.com/Kitware/VolView/blob/main/docs/mouse_controls.md.

I changed the shortcut to add modifier, and modified mouse_controls.md

floryst commented on src/composables/actions.ts:
```diff
@@ -47,6 +48,19 @@ const changeSlice = (offset: number) => () => {
+
+const deleteCurrentImage = () => () => {
+  const datasetStore = useDatasetStore();
+  datasetStore.remove(datasetStore.primaryImageID!);
```
I'm all for updating `datasetStore.remove` so that we can pass in null image IDs without checking. That way we can drop the `!` operator here.

```
remove(id: string | null) {
  if (!id) return
  ...
}
```

lchauvin replied:
I added the changes

PaulHax commented:
I was trying to squash some of lchauvin's commits and rebase rather than merge on main... but messed up force pushing them back on this PR =/  

Guess we just use #713
